packages/devtools_app/lib/src/screens/app_size/app_size_table.dart

```diff
@@ -51,6 +59,7 @@ class AppSizeAnalysisTable extends StatelessWidget {
       keyFactory: (node) => PageStorageKey<String>(node.name),
       sortColumn: sortColumn,
       sortDirection: SortDirection.descending,
+      onToggleNode: (TreemapNode n) => controller.changeAnalysisRoot(n),
```

Comment:
we may be able to solve this in a simpler way by setting the selectionNotifier parameter to `controller.analysisRoot`.

in TreeTableState, we automatically expand the parents of the selected node: https://github.com/flutter/devtools/blob/d24491817645320c789a0db624364982982d5c37/packages/devtools_app/lib/src/shared/table.dart#L442